Modify Python installation order

nickrobison opened the pull request:
This allows us to upgrade the existing python install to Python3, and then install Python2, that way both can exist at the same time. Taken from a discussion on the hombrew [discourse](https://discourse.brew.sh/t/brew-install-python3-fails/1756/6).

This also includes a superfluous change to the gdal2-python formula, in order to trigger the travis build.

This should hopefully resolve the issues in #308.

3nids commented on travis/before_install.sh:
```diff
@@ -51,51 +51,59 @@ for f in ${CHANGED_FORMULAE};do
     # Set up Python .pth files
     # get python short version (major.minor)
-    PY_VER=$(${HOMEBREW_PREFIX}/bin/python2 -c "import sys;print('{0}.{1}'.format(sys.version_info[0],sys.version_info[1]).strip())")
+    PY_VER=$(${HOMEBREW_PREFIX}/bin/python3 -c "import sys;print('{0}.{1}'.format(sys.version_info[0],sys.version_info[1]).strip())")
```
I think this should be python (without 3)

nickrobison replied:
The un-versioned python commands are linked to /usr/local/opt/python/libexec/bin, so we would either need to change the path, or leave the 3 suffix. the python command is simply symlinked to python3.

nickrobison commented:
I can confirm that the changes to the before_install.sh file resolve the issues @3nids was experiencing when trying to merge #308. Unfortunately, the build is failing because the current gdal2 bottle has some linking issues, see [here](https://travis-ci.org/OSGeo/homebrew-osgeo4mac/builds/362792721).

I've reverted my GDAL-python changes and squashed everything down to a single commit. Perhaps we could merge these changes and pull in #335 and #336 to rebuild the bottle and resolve the linking issues?

3nids commented:
awesome @nickrobison
